docs/developing/README.md

```diff
@@ -1,106 +1,33 @@
-The web component ecosystem is evolving fast, with many libraries and solutions appearing. High level frameworks such as [Angular](https://angular.io/guide/elements), [Vue](https://github.com/vuejs/vue-web-component-wrapper) and [Stencil](https://stenciljs.com/) provide solutions which compile to web components.
 
-## Browser standards
-We strongly believe that staying close to browser standards will be the best long term investment for your code. It is the basis for all our recommendations, and it means that sometimes we will not recommend a popular feature or functionality. It also means we can be faster to adopt and recommend new browser standards.
+## Going buildless
+Browsers have improved a lot over the past years. It's now possible to do web development without requiring any build tools, using the native module loader of the browser. We think this is a great fit for web components, and we recommend this as a general starting point.
 
-## lit-html
-While there are other good libraries out there, to get started we recommend using the `lit-html` library with the `LitElement` base class for developing web components.
+Build tools can quickly add a lot of complexity to your code, and make your code reliant on a specific build setup. We think it's best to avoid them during development, or only add them for light transformations if you know what you're doing.
 
-Check out the official documentation for: [lit-html](https://lit-html.polymer-project.org/) and [lit-element](https://lit-element.polymer-project.org/) or check out our code-samples below.
+Read [this article](https://dev.to/open-wc/developing-without-a-build-1-introduction-26ao) to learn more about this approach.
 
-Follow these steps to get started:
-1. Install the required dependencies:
-```
-npm i -D es-dev-server lit-element
-```
+## Development server
+We created [es-dev-server](https://open-wc.org/developing/es-dev-server.html) to help developing without build tools.
 
-2. Create a `index.html`
-```html
-<!doctype html>
-<html>
-  <head>
-    <script type="module" src="./src/my-component.js"></script>
-    </head>
-  <body>
-    <my-component></my-component>
-  </body>
-</html>
-```
+## Web component libraries
+You can write web components using just the basic web component APIs. This can be a great choice when you're looking to keep dependencies low. But generally, we recommend using lightweight libraries to help improve the developer experience and reduce boilerplate.
 
-3. Create a `my-component.js`
-```javascript
-import { LitElement, html } from 'lit-element';
+We recommend [lit-html](https://www.npmjs.com/package/lit-html) with the [lit-element](https://www.npmjs.com/package/lit-element) base class as a general-purpose library for building web components. `lit-html` is feature complete, extremely lightweight and offers a great development experience. Check out the [lit-html page](https://open-wc.org/developing/lit-html.html) for code examples and more information.
```

Comment:
The link to the 'lit-html page' doesnt work in the preview

Also, should we consider having a header for the demos? I feel like the demo's are a bit harder to find now, and they're one of the things that people find really useful and should be easy to find

Reply:
good idea, let's create a code examples page.

I don't use relative links so that they still work in github/npm. Though this page doesn't show up there...